README.md

```diff
@@ -11,7 +11,15 @@ and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/privacy-pass/)
 (v48+). An example server implementation that is compatible with this extension
 is available [here](https://github.com/privacypass/challenge-bypass-server).
 
-The protocol we use is based on a realization of a 'Verifiable, Oblivious
+In effect, the extension enables users to less frequently redo CAPTCHAs _from sites
```

Comment:
This paragraph does not belong up here. If it is necessary to add then it should be in the FAQ since this has only arisen from questions that have been asked in the issues.

Reply:
Also:

> The extension doesn't show if the protocol is
active or not. Not all users or websites use this, also since this technology is not
yet well known. Best chance to try this out is if you're in a region that are deemed to
need CAPTCHAs, and visit a CloudFlare site, like the example http://captcha.website/
If, by visiting, you don't get any CAPTCHA, you're welcome to hang around anyway and
learn more about this technology!

appear out of place in this repository. I can see the value in adding a clarifier to the FAQ that there are some issues with https://captcha.website that prevent it from working properly, but the rest seems unnecessary.